src/components/Accordion/__test__/accordion.spec.js

```diff
@@ -0,0 +1,399 @@
+import AccordionSection from './../../AccordionSection';
+
+describe('<Accordion />', () => {
+    it('should set the right "state" when onToggleSection is passed and it is not a function', () => {
+        const component = mount(
+            <Accordion
+                onToggleSection="string">
+                <AccordionSection
+                    name="accordion-test-1"
+                    label="Rainbow Accordion">
+                    AccordionSection-1
+                </AccordionSection>
+            </Accordion>,
+        );
+        const buttonComponent = component.find('button.rainbow-accordion-section_summary-button-heading');
+        buttonComponent.simulate('click');
+
+        expect(component.state('activeNames')).toEqual('accordion-test-1');
+    });
+    it('should fire an event with the right arguments when click in the AccordionSection and multiple is not passed', () => {
+        const handleToggleSectionMockFn = jest.fn();
+        const component = mount(
+            <Accordion
+                onToggleSection={handleToggleSectionMockFn}>
+                <AccordionSection
+                    name="accordion-test-1"
+                    label="Rainbow Accordion">
+                    AccordionSection-1
+                </AccordionSection>
+            </Accordion>,
+        );
+        const buttonComponent = component.find('button.rainbow-accordion-section_summary-button-heading');
+        buttonComponent.simulate('click');
+
+        expect(handleToggleSectionMockFn).toHaveBeenCalledWith(expect.any(Object), 'accordion-test-1');
+    });
+    it('should call handleToggleSection with correct "argument" when click in the "second AccordionSection" and "multiple" is passed', () => {
+        const handleToggleSectionMockFn = jest.fn();
+        const activeNames = ['accordion-test-1'];
+        const component = mount(
+            <div>
+                <Accordion
+                    multiple
+                    activeSectionNames={activeNames}
+                    onToggleSection={handleToggleSectionMockFn}>
+                    <AccordionSection
+                        name="accordion-test-1"
+                        label="Rainbow Accordion">
+                        AccordionSection-1
+                    </AccordionSection>
+                    <AccordionSection
+                        name="accordion-test-2"
+                        label="Rainbow Accordion">
+                        AccordionSection-2
+                    </AccordionSection>
+                </Accordion>
+            </div>,
+        );
+        const secondAccordionSection = component.find('ul').childAt(1);
+        const secondAccordionSectionButton = secondAccordionSection.find('button.rainbow-accordion-section_summary-button-heading');
+        secondAccordionSectionButton.simulate('click');
+
+        expect(handleToggleSectionMockFn.mock.calls[0][1]).toEqual(['accordion-test-1', 'accordion-test-2']);
```

Comment:
why not expect(handleToggleSectionMockFn).toHaveBeenCalledWith(expect.any(Object), ['accordion-test-1', 'accordion-test-2']);